docs/glossary.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Glossary
+sidebar_position: 90
+title: Glossary
+description: Kubewarden glossary
+keywords: [kubewarden, glossary]
+doc-persona: [kubewarden-all]
+doc-type: [explanation]
+doc-topic: [glossary]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/glossary"/>
+</head>
+
+## A
+
+### AdmissionPolicy
+
+A namespace-wide resource. The policy processes only those requests targeting
+the namespace in which the AdmissionPolicy is defined.
+
+## C
+
+### ClusterAdmissionPolicy
+
+A ClusterAdmissionPolicy defines how policies evaluate requests.
+
+### ClusterPolicyReport
+
+A [PolicyReport](#policyreport) and a ClusterPolicyReport store results of
+policy scans. Which one is used depends on the scope of the resource.
+
+## K
+
+### kwctl
+
+A CLI tool allowing administrators to test policies before applying them to a
+cluster.
+
+## M
+
+### MutatingWebhookConfiguration
```

Comment:
This is the [Kubernetes resource](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks) created by the Kubewarden controller to let Kubernetes knows where it should sent the `AdmissionReview`. In other words, this is how Kubewarden controller teach Kubernetes where to find the policy which could mutate resources before deploy them in the cluster